config.h

```diff
@@ -16,7 +16,7 @@ settings_t defaults = {
 .icons = { "dialog-information", "dialog-information", "dialog-warning" }, /* low, normal, critical */
 
 .transparency = 0,           /* transparency */
-.geom = "0x0",               /* geometry */
+.geometry = { 0 },           /* geometry */
```

Comment:
IMHO, we shouldn't just set it to `0` to successfully null it in the settings. IMHO the settings should get receive an elaborated initialisation.

Reply:
That's correct. Yes. But my intention was to split it off for better readability.